### DISCUSSION TITLE
how to use

### DISCUSSION THREAD

**Discussion by Kaiser-DMr** [ddosify/ddosify#52] [Q&A] [answered]
Hello Since I have not used go test, I am not very good at calling this tool. can you explain how i should use it
![AD9ECCC4-0C38-41EF-8A6E-0D31E0506AF1](https://user-images.githubusercontent.com/58213021/154991740-319dc4e5-d7d2-4331-9896-b50d62fc8794.jpeg)

**Comment by fatihbaltaci** [comment, marked as answer]
Hi @Kaiser-DMr, the installation guide is here: https://github.com/ddosify/ddosify#installation

How did you install ddosify?

**Reply by Kaiser-DMr** [reply]
I am using the Linux script to install the file seems to be there, but I use to view the version information but not

**Reply by Kaiser-DMr** [reply]
![Uploading 4A91D1E3-42CA-4D43-B910-21DAEBD496A1.jpeg…]()

**Reply by Kaiser-DMr** [reply]
![E1F714C7-FE4A-4F37-891E-A81DEEBF309B](https://user-images.githubusercontent.com/58213021/154996513-5ba430f8-b9d0-4cb2-a71c-a1be79d96217.jpeg)

**Reply by fatihbaltaci** [reply]
Is your system Ubuntu? Could you try these commands?

```
wget https://github.com/ddosify/ddosify/releases/latest/download/ddosify_amd64.deb
sudo dpkg -i ddosify_amd64.deb
```

**Reply by Kaiser-DMr** [reply]
No, my system is centos. I use a script to download and install

**Reply by Kaiser-DMr** [reply]
I solved it with docker. Thank you
